el-Joft/Recipe-app
# Create recipe CRUD endpoint

#### What does this PR do?
- Create the CRUD endpoints for recipe
#### Description of Task to be completed?
- Endpoints
```
- POST - http://127.0.0.1:8000/api/v1/ingredients/
- POST - http://127.0.0.1:8000/api/v1/steps/
- POST - http://127.0.0.1:8000/api/v1/recipe/
- GET - http://127.0.0.1:8000/api/v1/recipe/<id>
- GET - http://127.0.0.1:8000/api/v1/recipe/user/<userId>
- DELETE - http://127.0.0.1:8000/api/v1/recipe/<id>
```

#### How should this be manually tested?
- clone this repo
- checkout to `ft-recipe-crud`
- run `python manage.py migrate`
- test the endpoints
```
create ingredients
{
	"ingd_text": "Assorted beef"
}
create steps
{
	"text_step": "Boil the meat"
}
create recipe
{
	"name": "Check",
	"ingredient": ["936bbe47-a96a-457c-b3cf-f50d3cafe0ef"],
	"step": ["48ff2f4f-d3b3-4e77-8375-85fa02182f49"],
	"user": "0fb7a6e2-65ff-43c0-94af-52debd405d9f"
}

```
